packages/combobox/src/Combobox/Combobox.tsx

```diff
@@ -110,7 +113,8 @@ export function Combobox<M extends boolean>({
   size = ComboboxSize.Default,
   darkMode: darkModeProp,
   state = 'none',
-  errorMessage,
+  errorMessage = 'This input needs your attention',
```

Comment:
should these be defined as consts in our formfield component and used everywhere?

Reply:
sweet I added/exported the constant here: https://github.com/mongodb/leafygreen-ui/pull/2318/commits/b1782f71c9127e32f131467c77beaaa83952a5bd#diff-8389ce2a895700c252b7db5ddfbd3b5bb612b9e7458a82c1db6342a4b0f72d67R10-R13